sdk/keyvault/azure-keyvault-certificates/README.md

```diff
@@ -234,10 +234,13 @@ credential = DefaultAzureCredential()
 
 certificate_client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
 
-deleted_certificate = certificate_client.begin_delete_certificate("cert-name")
+deleted_certificate_poller = certificate_client.begin_delete_certificate("cert-name")
 
+deleted_certificate = deleted_certificate_poller.result()
 print(deleted_certificate.name)
 print(deleted_certificate.deleted_on)
+
+deleted_certificate_poller.wait()
```

Comment:
We don't show this in the other readmes ([for example](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-secrets/README.md#delete-a-secret)). I recall the reasoning there was that `wait` is only relevant to someone who immediately calls `purge`, and we didn't want to make this snippet more complex or to encourage users to always `wait`.

Reply:
sg, will get rid of it